tests/setups.py

```diff
@@ -0,0 +1,92 @@
+import os
+from functools import wraps
+import decorator
+import logging
+
+import boto3
+
+class ObjectGen:
```

Comment:
What purpose does this serve? Could you use a dictionary? If a class is necessary, checkout https://docs.python.org/3/library/types.html#types.SimpleNamespace

Reply:
The object is to mimic a regular python object whose members will be accessible using f=dot notation. I looked at SimpleNamespace and it is not obvious how it would help

Reply:
Update: I was able to use this, and will be making this change.